packages/components/src/RadioItem.tsx

```diff
@@ -50,6 +53,7 @@ const RadioItem = React.forwardRef<HTMLInputElement, RadioItemProps>(
       name,
       onChange,
       value,
+      dataTestid,
```

Comment:
As per my above comment,
```suggestion
      'data-testid': dataTestId,
```